deploy-manage/monitor/autoops/cc-connect-self-managed-to-autoops.md

```diff
@@ -0,0 +1,124 @@
+---
+applies_to:
+  deployment:
+    self:
+navigation_title: Connect your self-managed cluster
+---
+
+# Connect your self-managed cluster to AutoOps
+
+To use AutoOps with your self-managed cluster, you first need to create an {{ecloud}} account or log in to your existing account. Once you choose to connect AutoOps to your self-managed cluster, an installation wizard will guide you through the steps of installing Elastic Agent to send metrics from your self-managed cluster to AutoOps in {{ecloud}}.  
+
+Complete the steps in the following subsections to connect your cluster to AutoOps. The setup only takes about 10 minutes.
+
+## Prerequisites
+
+Ensure you meet the following requirements before proceeding:
+
+* You have an [Enterprise self-managed license](https://www.elastic.co/subscriptions) or an active self-managed [free trial](https://cloud.elastic.co/registration).
+* Your cluster is on a [supported {{es}} version](https://www.elastic.co/support/eol).
+* The agent you install for the connection is allowed to send metrics outside your organization to {{ecloud}}.
+* You have a dedicated user with following permissions to set up the Elastic Agent:
+
+    | Setting | Privileges |
+    | --- | --- |
+    | Cluster privileges | `monitor`, `read_ilm`, and `read_slm` |
+    | Index privileges | `*` indices: `monitor`, `view_index_metadata` <br><br> `.kibana*` indices: `read`  |
+
+## Connect to AutoOps
+
+:::{note}
+:::{include} /deploy-manage/monitor/_snippets/single-cloud-org.md
+:::
+:::
+
+1. Go to your {{ecloud}} home page.
+    * If you already have an {{ecloud}} account, log in to [{{ecloud}}](https://cloud.elastic.co?page=docs&placement=docs-body). 
+    * If you don’t have an {{ecloud}} account, [sign up](/deploy-manage/deploy/elastic-cloud/create-an-organization.md) and create an organization.
+2. In the **Connected clusters** section, select **Connect self-managed cluster**. 
+3. On the **Cloud Connected Services** page, under **AutoOps**, select **Connect**.
```

Comment:
```suggestion
3. On the **Cloud Connected Services** page, in the **AutoOps** section, select **Connect**.
```
As part of our accessibility guidance we avoid directional terms since they don't apply for people using screen readers.